in_toto/cmd/run.go

```diff
@@ -71,4 +76,8 @@ command is executed. Symlinks are followed.`)
 		`Paths to files or directories, whose paths and hashes
 are stored in the resulting link metadata after the
 command is executed. Symlinks are followed.`)
+	runCmd.PersistentFlags().StringVarP(&certPath,
+		"cert", "c", "",
```

Comment:
Should this be a []String?

Reply:
I don't think it makes sense for multiple certs to be passed in here, does it?  This cert should certify the private key passed in 1:1